### PR TITLE
fix(FEC-10583): fix linear image ads

### PR DESCRIPTION
**Pull request by SivanA-Kaltura**
### Description of the Changes

Prevent display of bottom bar controls in an ad in case it's not a video ad.
Fixes FEC-10583.

Related PR: https://github.com/kaltura/playkit-js-ima/pull/209

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/ui-presets/ads.js [hunk 1]
```diff
@@ -64,20 +64,22 @@ function AdsUI(props: any, context: any): ?React$Element<any> {
...
-                  </Fragment>
-                }
-                rightControls={
-                  <Fragment>
-                    <Volume />
-                    <Fullscreen />
-                  </Fragment>
-                }
-              />
+              {displayBottomBar(props) ? (
```
Comment (yairans) [inline src/ui-presets/ads.js:67]: the full screen component should be injected for image also no?

File: src/components/active-preset/active-preset.js [hunk 2]
```diff
@@ -16,6 +16,8 @@ const mapStateToProps = state => ({
     shell: state.shell,
     engine: {
       adBreak: state.engine.adBreak,
+      adIsLinear: state.engine.adIsLinear,
+      adContentType: state.engine.adContentType,
```
Comment (yairans) [inline src/components/active-preset/active-preset.js:20]: I would change ads preset to be a component (like playback for example) and use these states there

File: src/ui-presets/ads.js [hunk 3]
```diff
@@ -86,7 +100,22 @@ function AdsUI(props: any, context: any): ?React$Element<any> {
...
+  state: {
+    shell: state.shell,
+    engine: {
+      adIsLinear: state.engine.adIsLinear,
+      adContentType: state.engine.adContentType
+    }
+  }
+});
+
+const AdsUIComponent = connect(mapStateToProps)(withKeyboardEvent(PRESET_NAME)(AdsUI));
```
Comment (yairans) [inline src/ui-presets/ads.js:118]: possible to move to decorator syntax?

File: src/ui-presets/ads.js [hunk 4]
```diff
@@ -86,7 +100,22 @@ function AdsUI(props: any, context: any): ?React$Element<any> {
...
 
-const AdsUIComponent = withKeyboardEvent(PRESET_NAME)(AdsUI);
+/**
+ * mapping state to props
+ * @param {*} state - redux store state
+ * @returns {Object} - mapped state to this component
+ */
+const mapStateToProps = state => ({
+  state: {
+    shell: state.shell,
```
Comment (yairans) [inline src/ui-presets/ads.js:110]: remove if not in use

File: src/ui-presets/ads.js (hunk 3 above)
Comment (SivanA-Kaltura) [reply src/ui-presets/ads.js:118]: decorator must be on a class

File: src/ui-presets/ads.js (hunk 4 above)
Comment (SivanA-Kaltura) [reply src/ui-presets/ads.js:110]: used to get value of shell.isMobile

**Review by yairans — APPROVED** [review]

**Pull request merged**